Ka'het Engines (#5501)
endless-sky/endless-sky: data/ka'het/ka'het outfits.txt

# Engines:

outfit "Maeri Engine Nacelles"
	category "Engines"
	"cost" 928000
	thumbnail "outfit/ka'het maeri engine"
	"mass" 52
	"outfit space" -76
	"engine capacity" -59
	"energy consumption" 4.21
	"heat generation" 1.1
	"thrust" 35.1
	"thrusting heat" 1.8
	"turn" 821.4
	"turning heat" 0.9
	"flare sprite" "effect/ka'het flare/large"
		"frame rate" 30
	"flare sound" "plasma medium"
	description "The Ka'het engines are peculiar in that they need to be continously charged to with large amounts of energy: while this means that less power remains available for other subsystems, it is somewhat more efficient than most types of propulsions at doing so, and generates only a tiny amount of heat."

outfit "Ka'het Sustainer Nacelles"
	category "Engines"
	"cost" 357000
	thumbnail "outfit/ka'het sustainer engine"
	"mass" 27
	"outfit space" -34
	"engine capacity" -27
	"heat generation" 0.6
	"energy consumption" 2.39
	"thrust" 13.2
	"thrusting heat" 1.8
	"turn" 432.1
	"turning heat" 0.4
	"flare sprite" "effect/ka'het flare/medium"
		"frame rate" 30
	"flare sound" "plasma small"

> 	description "While the Ka'Het vessels are already equipped with proper engines, most of them use a sustainer engine in addition to give the ship enough manuevrability to be useful in combat. Following the same design as its bigger cousins, the sustainer needs constant energy to maitain its thrusters and steerings ready."

Reviewer: ```suggestion
	description "While the Ka'Het vessels are already equipped with proper engines, most of them use a sustainer engine in addition to give the ship enough maneuverability to be useful in combat. Following the same design as its bigger cousins, the sustainer needs constant energy to maitain its thrusters and steerings ready."
```

	"outfit space" -3
	"energy capacity" 600
	description "This tiny battery was developed for use by the Ka'mar drones as an emergency energy storage, in case their generators can't quite suffice for the high power demand of most Builder systems."

# Engines:

outfit "Maeri Engine Nacelles"
	category "Engines"
	"cost" 928000
	thumbnail "outfit/ka'het maeri engine"
	"mass" 52
	"outfit space" -76

> 	"engine capacity" -59

Reviewer 2: 52 mass, but 59 engine cap? The other engines here have a matching mass and engine capacity.

Author: fixed the mass

	"energy consumption" 5.01
	"heat generation" 1.1
	"thrust" 37.1
	"thrusting heat" 2.8
	"turn" 931.4
	"turning heat" 1.3
	"flare sprite" "effect/ka'het flare/large"
		"frame rate" 30
	"steering flare sprite" "effect/ka'het flare/large"
		"frame rate" 30
	"flare sound" "plasma medium"

> 	description "The Ka'het engines are peculiar in that they need to be continously charged with large amounts of energy: while this means that less power remains available for other subsystems, it is somewhat more efficient than most types of propulsions at doing so, and generates only a tiny amount of heat."

Reviewer 3: ```suggestion
	description "The Ka'het engines are peculiar in that they need to be continuously charged with large amounts of energy: while this means that less power remains available for other subsystems, it is somewhat more efficient than most types of propulsions at doing so, and generates only a tiny amount of heat."
```

> 	description "The Ka'het engines are peculiar in that they need to be continously charged with large amounts of energy: while this means that less power remains available for other subsystems, it is somewhat more efficient than most types of propulsions at doing so, and generates only a tiny amount of heat."

Reviewer 3: > at doing so

At doing what? You might consider clarifying here or rewording in a way that makes it more clear what these engines are more efficient at.

Reviewer 4: I imagine it refers to charging, but yes, this needs to be reworded/clarified.

Author: reworded

outfit "Ka'het Sustainer Nacelles"
	category "Engines"
	"cost" 357000
	thumbnail "outfit/ka'het sustainer engine"
	"mass" 27
	"outfit space" -34
	"engine capacity" -27
	"heat generation" 0.6
	"energy consumption" 2.49
	"thrust" 15.1
	"thrusting heat" 1.0
	"turn" 432.1
	"turning heat" 0.5
	"flare sprite" "effect/ka'het flare/medium"
		"frame rate" 30
	"steering flare sprite" "effect/ka'het flare/medium"
		"frame rate" 30
	"flare sound" "plasma small"

> 	description "While the Ka'Het vessels are already equipped with proper engines, most of them use a sustainer engine in addition to give the ship enough maneuverability to be useful in combat. Following the same design as its bigger cousins, the sustainer needs constant energy to maintain its thrusters and steerings ready."

Reviewer 2: ```suggestion
	description "While Ka'het vessels are already equipped with proper engines, most of them use a sustainer engine in addition to give the ship enough maneuverability to be useful in combat. Following the same design as its bigger cousins, the sustainer needs constant energy to maintain its thrusters and steering."
```

> 	description "While the Ka'Het vessels are already equipped with proper engines, most of them use a sustainer engine in addition to give the ship enough maneuverability to be useful in combat. Following the same design as its bigger cousins, the sustainer needs constant energy to maitain its thrusters and steerings ready."

Reviewer 3: ```suggestion
	description "While the Ka'Het vessels are already equipped with proper engines, most of them use a sustainer engine in addition to give the ship enough maneuverability to be useful in combat. Following the same design as its bigger cousins, the sustainer needs constant energy to maintain its thrusters and steerings ready."
```

outfit "Telis Engine Nacelles"
	category "Engines"
	"cost" 1372000
	thumbnail "outfit/ka'het telis engine"
	"mass" 81
	"outfit space" -109
	"engine capacity" -81
	"energy consumption" 6.81
	"heat generation" 1.4
	"thrust" 55.4
	"thrusting heat" 3.5
	"turn" 1379.2
	"turning heat" 1.9
	"flare sprite" "effect/ka'het flare/large"
		"frame rate" 30
	"steering flare sprite" "effect/ka'het flare/large"
		"frame rate" 30
	"flare sound" "plasma medium"

> 	description "While its size is not impressive compared to other engines, these nacelles that used to be the entire engine block of a Telis'het if used in companion with a sustainer or two can be a powerful form of propulsion, combining both turning and thrusting engines in one."

Reviewer 3: ```suggestion
	description "While their size is not impressive compared to other engines, these nacelles, which used to be the entire engine block of a Telis'het, if used in companion with a sustainer or two, can be a powerful form of propulsion, combining both turning and thrusting engines in one."
```
OR
```suggestion
	description "These nacelles used to comprise the entire engine block of a Telis'het. While their size is not impressive compared to other engines, if used in conjunction with a sustainer or two, they can be a powerful form of propulsion, combining both turning and thrusting engines in one."
```
Top is simply grammar corrections, bottom is some style/wording suggestions. Your choice.

Author: The last added comma seems unnecessary in both, especially in the first suggestion

Reviewer 3: The bottom suggestion changed it into a traditional [conditional sentence](https://www.grammarly.com/blog/conditional-sentences/#:~:text=Punctuating%20Conditional%20Sentences&text=Here's%20the%20skinny%3A-,Use%20a%20comma%20after%20the%20if%2Dclause%20when%20the%20if,if%20I'd%20had%20time.) (See: Punctuating Conditional Sentences, at the very bottom of the article). The if-clause is `if used in conjunction with a sustainer or two`, and the main clause is `they can be a powerful form of propulsion`.
Your original is strange because you've inserted your if-clause between the main clause: the noun, `nacelles`, precedes the if-clause, but the rest of the clause comes after: `can be a powerful form of propulsion`. I'm not really sure how to handle that, but my best guess would be because it's interrupting the phrase, it needs commas separating it. Does that make sense?

> outfit "Ka'het Compact Engine"

Reviewer 2: Should this be Ka'mar instead of Ka'het?

Author: The MHD and support battery are still called "Ka'het" as the rest of their stuff, so to avoid confusion I'd rather call them all Ka'het

	category "Engines"
	"cost" 172000
	thumbnail "outfit/ka'het compact engine"
	"mass" 21
	"outfit space" -26
	"engine capacity" -21
	"heat generation" 0.3
	"energy consumption" 0.48
	"thrust" 11.2
	"thrusting energy" 1.7
	"thrusting heat" 1.2
	"turn" 234.4
	"turning energy" 0.42
	"turning heat" 0.3
	"flare sprite" "effect/ka'het flare/medium"
		"frame rate" 30
	"steering flare sprite" "effect/ka'het flare/medium"
		"frame rate" 30
	"flare sound" "plasma small"

> 	description "This tiny engine was developed as the main propulsion of the Ka'mar drones, long before the Ka'het were created and their nacelle design standardized. Its improvements over the time allowed it to mantain a comparable efficency to the larger models, although unlike those it requires both a constant amount of energy and some more to activate the engines."

Reviewer 3: ```suggestion
	description "This tiny engine was developed as the main propulsion of the Ka'mar drones, long before the Ka'het were created and their nacelle design standardized. Its improvements over time allowed it to maintain a comparable efficiency to the larger models, although unlike those it requires both a constant amount of energy and some more to activate the engines."
```

> 	description "This tiny engine was developed as the main propulsion of the Ka'mar drones, long before the Ka'het were created and their nacelle design standardized. Its improvements over time allowed it to maintain a comparable efficiency to the larger models, although unlike those it requires both a constant amount of energy and some more to activate the engines."

Reviewer 2: ```suggestion
	description "This tiny engine was developed as the main propulsion of the Ka'mar drones, long before the Ka'het were created and their nacelle design standardized. Improvements over time allowed it to maintain a comparable efficiency to the larger models, although unlike those, it requires some energy to activate the engines in addition to the constant energy drain that all Builder-designed engines have."
```
